frontend/src/app/main/data/workspace.cljs

```diff
@@ -233,6 +233,19 @@
                                   (->> (filter (comp t/pointer? val) data)
                                        (resolve-pointers id)
                                        (rx/map #(update file :data merge %)))))
+                               (rx/mapcat
```

Comment:
Now when loading libraries need to load all the pages, and resolve the fragments too (previously only assets were loaded)